frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseSyncToParentByMergeAction.java

```diff
@@ -76,20 +78,20 @@ protected void onUpdate(AnActionEvent anActionEvent) {
   public void actionPerformed(AnActionEvent anActionEvent) {
 
     val project = getProject(anActionEvent);
-    val gitRepository = getSelectedGitRepository(anActionEvent).getOrNull();
-    val stayingBranchName = getNameOfBranchUnderAction(anActionEvent).getOrNull();
+    val gitRepository = getSelectedGitRepository(anActionEvent);
+    val stayingBranchName = getNameOfBranchUnderAction(anActionEvent);
     if (gitRepository == null || stayingBranchName == null) {
       return;
     }
 
-    val movingBranch = getManagedBranchByName(anActionEvent, stayingBranchName).getOrNull();
+    val movingBranch = getManagedBranchByName(anActionEvent, stayingBranchName);
     if (movingBranch == null) {
       return;
     }
     // This is guaranteed by `syncToParentStatusDependentActionUpdate` invoked from `onUpdate`.
     assert movingBranch.isNonRoot() : "Branch that would be merged INTO is a root";
 
-    val currentBranchName = Option.of(gitRepository.getCurrentBranch()).map(b -> b.getName()).getOrNull();
+    val currentBranchName = Option.of(gitRepository.getCurrentBranch()).map(GitReference::getName).getOrNull();
```

Comment:
Out of curiosity, was there a specific reason to use method reference instead of lambda (like, Checker Framework reporting some errors on the latter)? or just a matter of code style?

Reply:
Just a matter of code style, it made it more readable imho